features/account/registering.feature

```diff
@@ -50,3 +63,16 @@ Feature: Account registration
         When I register with email "ghastly@bespoke.com" and password "suitsarelife"
         Then I should be notified that new account has been successfully created
         And a welcoming email should have been sent to "ghastly@bespoke.com"
+
+    @ui @email
+    Scenario: Receiving a verification email after registration
+        When I register with email "ghastly@bespoke.com" and password "suitsarelife"
+        Then I should be notified that new account has been successfully created
+        And a verification email should have been sent to "ghastly@bespoke.com"
```

Comment:
This scenario is already covered (see [here](https://github.com/michalmarcinkowski/Sylius/blob/0ce1d75d208bdbd265199d04f5c7939318d94f21/features/account/verification/verifying_email_address.feature#L49-L49)).